### PR TITLE
Update of the section 4

### PR DESCRIPTION
**Pull request by bkhelifi**

### CODE REVIEW

**Comment by mservillat** [comment]
Need to change "A&A" to "A\&A", line 145 of .bib

**Comment by bkhelifi** [comment]
@mservillat, do you mean "A&A" to "A\&A"?

**Comment by mservillat** [comment]
oups, "A\\&A"

File: VOHE-Note.bib [hunk 1]
```diff
@@ -137,11 +137,27 @@ @misc{std:W3CProvDM
...
-      year={2023},
-      eprint={2309.03007},
-      archivePrefix={arXiv},
-      primaryClass={astro-ph.HE}
+@article{giunti2022,
+	author = {{Giunti, L.} and {Acero, F.} and {Khélifi, B.} and {Kosack, K.} and {Lemière, A.} and {Terrier, R.}},
+	title = {Constraining leptonic emission scenarios for the PeVatron candidate HESS J1702−420 with deep XMM-Newton observations⋆},
+	DOI= "10.1051/0004-6361/202244696",
+	url= "https://doi.org/10.1051/0004-6361/202244696",
+	journal = {A&A},
```
Comment (bkhelifi) [inline VOHE-Note.bib]:
```suggestion
	journal = {A\&A},
```

File: VOHE-Note.bib [hunk 2]
```diff
@@ -137,11 +137,27 @@ @misc{std:W3CProvDM
...
-      year={2023},
-      eprint={2309.03007},
-      archivePrefix={arXiv},
-      primaryClass={astro-ph.HE}
+@article{giunti2022,
+	author = {{Giunti, L.} and {Acero, F.} and {Khélifi, B.} and {Kosack, K.} and {Lemière, A.} and {Terrier, R.}},
+	title = {Constraining leptonic emission scenarios for the PeVatron candidate HESS J1702−420 with deep XMM-Newton observations⋆},
+	DOI= "10.1051/0004-6361/202244696",
+	url= "https://doi.org/10.1051/0004-6361/202244696",
+	journal = {A\&A},
```
Comment (mservillat) [inline VOHE-Note.bib]: well, not the issue, but you can use \\aap for A&A

File: VOHE-Note.bib [hunk 3]
```diff
@@ -137,11 +137,27 @@ @misc{std:W3CProvDM
...
-@misc{unbehaun2023prospects,
-      title={Prospects for combined analyses of hadronic emission from $\gamma$-ray sources in the Milky Way with CTA and KM3NeT},
-      author={T. Unbehaun, L. Mohrmann, et al.},
-      year={2023},
-      eprint={2309.03007},
-      archivePrefix={arXiv},
-      primaryClass={astro-ph.HE}
+@article{giunti2022,
+	author = {{Giunti, L.} and {Acero, F.} and {Khélifi, B.} and {Kosack, K.} and {Lemière, A.} and {Terrier, R.}},
+	title = {Constraining leptonic emission scenarios for the PeVatron candidate HESS J1702−420 with deep XMM-Newton observations⋆},
```
Comment (mservillat) [inline VOHE-Note.bib]: I think the issue is here : there is a Unicode character : − (U+2212) to be replaced
Maybe remove the * at the end too

File: VOHE-Note.bib (hunk 2 above)
Comment (bkhelifi) [inline VOHE-Note.bib]:
```suggestion
	journal = {\aap},
```

File: VOHE-Note.bib (hunk 3 above)
Comment (bkhelifi) [reply VOHE-Note.bib]: Which is a typo, by the way...

Comment (bkhelifi) [inline VOHE-Note.bib]:
```suggestion
	title = {Constraining leptonic emission scenarios for the PeVatron candidate HESS J1702−420 with deep XMM-Newton observations},
```

**Pull request merged**